### ISSUE TITLE
Compilation error against Nimble devel

### ISSUE THREAD

**Issue by wizzardx** [Vladar4/libtcod_nim#5]
```
/home/david/.nimble/pkgs/libtcod_nim-0.98/libtcod_nim/private/console.nim(163, 14) Error: usage of 'isNil' is a user-defined error
```

**Comment by Vladar4** [comment]
There were many breaking changes introduced in the Nim v0.18.1 devel version. Until the v0.19.0 is released (and sdl1 package is fixed vs it) I propose you to use Nim v0.18.0 or older.

Later I will create `version0.99` branch to adapt the project to the upcoming Nim v0.19.0.

**Comment by Vladar4** [comment]
Fixed in v0.98.1
